Ginger/Ginger/UserControlsLib/UCTreeView/TreeView1.xaml

```diff
@@ -27,6 +27,10 @@
             </DockPanel>
         </DockPanel>
 
+        <DockPanel x:Name="xTitleSection2"  HorizontalAlignment="Stretch" LastChildFill="true" Margin="0,0,0,0" VerticalAlignment="Stretch" Visibility="Collapsed">
```

Comment:
Let's use the tree title itself instead of created new one. And control the styling from different places